sw/host/opentitantool/src/command/update_usr_access.rs

```diff
@@ -36,10 +36,11 @@ impl CommandDispatch for UpdateUsrAccess {
         _transport: &TransportWrapper,
     ) -> Result<Option<Box<dyn Annotate>>> {
         let mut bs = fs::read(&self.input)?;
-        usr_access_set(
-            &mut bs,
-            self.usr_access.unwrap_or_else(usr_access_timestamp),
-        )?;
+        let mut usr_access_val = self.usr_access;
```

Comment:
Super nit: could we just figure out the value of `usr_access_val` with a match expression?